player/public/coreplugins/org.ekstep.ecmlrenderer-1.0/renderer/qspatch.js

```diff
@@ -390,5 +390,71 @@ var qspatch = {
             'image': data.image ? data.image : undefined,
             'audio': data.audio ? data.audio : undefined,
         })
+    },
+    //Make default font size for question options
+    setDefaultFontSize: function(data){
+      if(data.stage['org.ekstep.questionset']){
+        try {
+          var questionSetData = data.stage['org.ekstep.questionset'];
+          _.each(questionSetData['org.ekstep.question'],function(item,key){
+              var questionData = JSON.parse(item.data.__cdata);
+              questionData.question.text = qspatch.changeFontSize(questionData.question);
+              item.data.__cdata = JSON.stringify(questionData);
+              item = qspatch.checkOptionsFontSize(item);
+              questionSetData['org.ekstep.question'][key] = item;
+          });
+          // Renderer.theme = new ThemePlugin(questionSetData);
+        } catch (e) {
+            console.log(e);
+        }
+      }
+    },
+    checkOptionsFontSize: function(item){
+      var questionItem = item;
+      var questionOptionsData = JSON.parse(item.data.__cdata);
+
+      switch (item.pluginId) {
+        case 'org.ekstep.questionunit.mcq':
+        case 'org.ekstep.questionunit.sequence':
+              questionOptionsData.options = qspatch.changeOptionsFontSize(questionOptionsData.options,item.type);
+              questionItem.data.__cdata = JSON.stringify(questionOptionsData);
+              break;
+        case 'org.ekstep.questionunit.mtf':
+              questionOptionsData.option.optionsLHS = qspatch.changeOptionsFontSize(questionOptionsData.option.optionsLHS,item.type);
+              questionOptionsData.option.optionsRHS = qspatch.changeOptionsFontSize(questionOptionsData.option.optionsRHS,item.type);
+              questionItem.data.__cdata = JSON.stringify(questionOptionsData);
+              break;
+        default: break;
+
+      }
+      return questionItem;
+    },
+    changeOptionsFontSize: function(options,type){
+      var optionsData = options;
+      if(type == 'mcq'){
+        _.each(options,function(option,key){
+          optionsData[key].text = qspatch.changeFontSize(option);
+        });
+      }else if(type == 'mtf'){
+        _.each(options,function(option,key){
+          optionsData[key].text = '<p style="font-size:1.28em">' + optionsData[key].text + '</p>';
+        });
+      }
+      return optionsData;
+    },
+    changeFontSize: function(data){
+      var index = data.text.indexOf("<p><span");
+
+      if(index == 0){
```

Comment:
Why still we are using index. Get fontSize first, if it is undefined, then do 
else part

Reply:
What if <p> tag itself is not there? How we have to get children and elements?